.circleci/config.yml

```diff
@@ -23,20 +23,16 @@ jobs:
         - restore_cache:
             keys:
             - v1-pip-dependencies-{{ checksum "requirements.txt" }}
```

Comment:
Now that only one key is present, consider moving from ```keys``` to ```key``` as mentioned [in the reference documentation for CircleCI](https://circleci.com/docs/2.0/configuration-reference/#restore_cache).